### PR TITLE
Add hri information

### PR DESCRIPTION
**Pull request by strieflin**

### CODE REVIEW

**Comment by strieflin** [comment]
@sbckr: Please review but **do not merge**, yet.

**Comment by sbckr** [comment]
@strieflin how about adding anchors to the contributor descriptions?

e.g. as follows:
- add anchors to `docs/community/ecosystem.md`
```markdown
# Carbyne Stack Ecosystem
<a name="bosch"/>
![Bosch Logo](../images/ecosystem/bosch-logo.png){ class="ecosystem-logo" }
[...]

---
<a name="hri"/>
![HRI Logo][...]

---
<a name="uts"/>
![UTS Logo][...]
```
- User anchors as link destination in `overrides/home.html`
```html
[...]
                        <h3 style="padding-top: 30px;">Contributors</h3>
                        <div class="used-by-logo-banner">
                            <a href="community/ecosystem/#bosch">
                               <img
                                src="images/ecosystem/bosch-logo.png"
                                class="used-by-logo"
                                alt="Bosch Logo"
                                draggable="false"
                                />
                            </a>
                            <a href="community/ecosystem/#hri">
                              <img
                                  src="images/ecosystem/hri-logo.png"
                                  class="used-by-logo"
                                  alt="HRI Logo"
                                  draggable="false"
                              />
                            </a>
                            <a href="community/ecosystem/#uts">
                              <img
                                  src="images/ecosystem/uts-logo.png"
                                  class="used-by-logo invert"
                                  alt="UTS Logo"
                                  draggable="false"
                              />
                            </a>
                        </div>
[...]
```

Might especially be interesting once there are more contributors to Carbyne Stack 😉

File: docs/community/ecosystem.md [hunk 1]
```diff
@@ -9,7 +9,20 @@ to conduct proof-of-concepts in multiple domains with various partners.
 
 ---
 
-![Image title](../images/uts-logo.png){ class="ecosystem-logo" }
+![HRI Logo](../images/ecosystem/hri-logo.png){ class="ecosystem-logo" }
```
[the remark this answers is not in the document]
Comment (strieflin) [reply docs/community/ecosystem.md:12]: Good catch! White logo was added later without checking the ecosystem page again. A pity that there are no automated test for such things 😄.

**Comment by strieflin** [comment]
Anchors have been implemented as proposed.

**Pull request merged**